src/lambda/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "snowplow-shared-consumer",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/main.js",
+  "scripts": {
+    "build": "rm -rf dist && tsc",
+    "watch": "tsc -w & nodemon",
+    "start": "node dist/main.js",
+    "start:dev": "npm run build && npm run watch",
+    "test-ci": "npm test",
+    "test:watch": "npm test -- --watchAll",
+    "test": "jest \"\\.spec\\.ts\"",
+    "test-integrations": "jest \"\\.integration\\.ts\" --forceExit",
+    "lint-check": "eslint --fix-dry-run \"src/**/*.ts\"",
+    "lint-fix": "eslint --fix \"src/**/*.ts\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Pocket/account-delete-monitor.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Pocket/shared-snowplow-consumer/issues"
+  },
+  "homepage": "https://github.com/Pocket/shared-snowplow-consumer#readme",
+  "dependencies": {
+    "@pocket-tools/apollo-utils": "^2.2.0",
+    "@sentry/serverless": "7.7.0",
+    "aws-xray-sdk-express": "^3.3.7",
+    "aws-lambda": "1.0.7",
+    "typescript": "^4.8.2"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.3.3",
+    "@types/jest": "^27.4.0",
+    "@typescript-eslint/eslint-plugin": "^5.10.0",
+    "@typescript-eslint/parser": "^5.10.2",
+    "chai": "^4.3.6",
+    "eslint": "^8.8.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "jest": "^27.4.7",
+    "nock": "^13.2.2",
+    "nodemon": "^2.0.15",
+    "prettier": "^2.5.1",
+    "ts-jest": "^27.1.3",
+    "ts-node": "^10.9.1"
+  }
```

Comment:
Should the lambda be using our ESLint/TS config setup, too? Just like it does in the root of the repository:

```json
  "@pocket-tools/eslint-config": "2.0.0",
  "@pocket-tools/tsconfig": "2.0.1",
```

Reply:
my understanding is the main package has lint covered for this folder too. so this would be repetitive right?